jitsi-utils-kotlin/src/main/kotlin/org/jitsi/utils/AtomicExtensions.kt

```diff
@@ -0,0 +1,26 @@
+/*
+ * Copyright @ 2018 - present 8x8, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.jitsi.utils
+
+import java.lang.Long.max
+import java.lang.Long.min
+import java.util.concurrent.atomic.AtomicLong
+
+fun AtomicLong.increaseAndGet(newValue: Long): Long
```

Comment:
I think these names are confusing.  I'd rather do something clear (albeit perhaps long-winded) like `AtomicLong.setToMaxOfCurrentOr(val: Long)` or something like that?

Reply:
I named them similar to `incrementAndGet`, but I can see how that can be confusing since the semantics are slightly different. How about `bumpTo` and `dropTo`?

Reply:
Maybe `minAssign` and `maxAssign` by analogy with `plusAssign` etc?